computer_assisted_intervention/video_seg.ipynb

```diff
@@ -369,7 +369,14 @@
    "source": [
     "## Normal training loop\n",
     "\n",
-    "A normal training loop will be excuted. Meanwhile, the iou value of each epoch will be recorded as the accuracy."
+    "A normal training loop will be excuted. Meanwhile, the iou value of each epoch will be recorded as the accuracy. \n",
+    "There is a pretrained bundle for surgical tool segmentation task. This model can be loaded to the network by adding codes to the block below:\n",
+    "```\n",
+    "pretrained_model = monai.bundle.load(\n",
```

Comment:
Suggest var name "pretrained_model" to "pretrained_weights".

Thanks.

Reply:
Hi @Nic-Ma ,
    The var name has been updated.
Thanks,
Bin